lib/linter-reek.js

```diff
@@ -77,7 +77,17 @@ export default {
           ignoreExitCode: true,
         };
 
-        const output = await helpers.exec(this.executablePath, [filePath], execOpts);
+        let output;
+        try {
+          output = await helpers.exec(this.executablePath, [filePath], execOpts);
+        } catch (e) {
+          if (e.message !== 'Process execution timed out') throw e;
+          atom.notifications.addInfo('linter-reek: Linter timed out', {
+            description: 'A timeout occured while executing reek, it could be due to lower resources ' +
+                         'or a temporary overload, try a few seconds later, please.',
```

Comment:
> ", try a few seconds later, please"

I think we should just cut this part out, since when the provider is called is determined by the thing calling it (and is usually automatic).